File: Android/tuilivekit/src/main/java/com/trtc/uikit/livekit/common/core/store/state/LiveState.java
package com.trtc.uikit.livekit.common.core.store.state;

import com.trtc.uikit.livekit.common.core.store.state.view.ViewState;
import com.trtc.uikit.livekit.common.core.store.state.operation.OperationState;

public class LiveState {
    public OperationState operationState = new OperationState();
    public ViewState      viewState      = new ViewState();
}

File: Flutter/livekit/lib/state/live_state.dart
import 'package:tencent_live_uikit/state/operation/operation_state.dart';
import 'package:tencent_live_uikit/state/view/view_state.dart';

class LiveState {
  final OperationState operationState = OperationState();
  final ViewState viewState = ViewState();

  void reset() {
    viewState.reset();
    operationState.reset();
  }
}
